runCompareTwoGenomesColinear.sh

```diff
@@ -16,23 +16,39 @@ if [ $# -ne 11 ]; then
         exit 1
 fi
 
-echo Running: $(basename $0) $1 $2 $3 $4 $5 $6 $7 $8 $9 ${10} ${11}
+if [ ${11} == '.' ]; then
+   if [ -z ${XM_FONTS+x} ]; then
+      echo "WARN: No font path defined and ENV variable XM_FONTS not found"
+   fi
+else
+   XM_FONTS=${11}
+fi
 
-# source PATH-TO-SOURCE (IF NEEDED)
+echo Running: $(basename $0) $1 $2 $3 $4 $5 $6 $7 $8 $9 ${10} ${XM_FONTS}
 
-echo "Make sure xmatchview.py, cross_match and minimap2 are in your PATH"
+FEATURE_OPTS=""
+if [ $8 != '.' ]; then
+   FEATURE_OPTS=" -y $8"
+fi
+if [ $9 != '.' ]; then
+   FEATURE_OPTS=" -y $9"
+fi
```

Comment:
I believe that to maintain the same behaviour as before, `$9` should be ` -e $9`. Also, looks like you are overwriting the `FEATURE_OPTS` variable if both `$8` and `$9` are specified?

Reply:
Yes, that's a silly mistake, fixed in recent push.